packages/terra-core-docs/package.json

```diff
@@ -84,7 +84,7 @@
     "terra-status": "^4.57.0",
     "terra-status-view": "^4.62.0",
     "terra-switch": "^1.12.0",
-    "terra-table": "^4.36.0",
+    "terra-table": "~/terra-core/packages/terra-table",
```

Comment:
Not sure if this is temporary or not but FYI, this will cause failures in terra-ui terra-table won't exist at this path.